docs/platform/concepts/console.rst

```diff
@@ -0,0 +1,22 @@
+Aiven web console
+=================
```

Comment:
Since we no longer have the separate console section, I don't think we need to have the landing page for the console either.

Reply:
Yep, removed ;)